### PR TITLE
basic teamplate for credit checkout page

### PR DESCRIPTION
**Pull request by zubair-arbi**
ECOM-1797

@awais786 @aamir-khan @ahsan-ul-haq 
Basic credit checkout landing page for one credit provider.
Page content is static.

### CODE REVIEW

**Comment by clintonb** [comment]
Please remove the custom styles and use the default Bootstrap styling: http://getbootstrap.com/.

File: ecommerce/static/sass/main.scss [hunk 1]
```diff
@@ -17,3 +17,7 @@
 // utility/misc.
 // --------------------
 @import 'developer';    // developer-friendly file: add rough/WIP styling that needs UI triage
+
+//// credit app styling
+//// --------------------
+@import 'credit';
```
Comment (clintonb) [inline ecommerce/static/sass/main.scss:23]: This doesn't actually work.

Comment (zubair-arbi) [inline ecommerce/static/sass/main.scss:23]: Sorry forgot to add a file.
@clintonb I am still working on this. Please wait until I make changes.
For consistency I fetched the minimal styling required from lms, is it good way?

Comment (clintonb) [inline ecommerce/static/sass/main.scss:23]: We will style Bootstrap to resemble the LMS styling. Please refrain from copying large swaths of LMS SASS and putting them here.

Comment (zubair-arbi) [inline ecommerce/static/sass/main.scss:23]: Ok. I will make changes accordingly and let you know.

**Comment by zubair-arbi** [comment]
@clintonb I have updated my PR to use bootstrap for creating credit checkout page (responsive design tested locally). Please let me know if my approach is correct.
@andywaldrop here is the screen shot (for single credit provider).
screen shot 2015-07-01 at 3 08 44 pm

**Comment by andywaldrop** [comment]
@zubair-arbi Please have another pass at the styles you're using.

Course image max width should be 140px
Partner logo max width should be 100px

Check margins against comps

Header copy 21px semibold line height 1.5 
Support copy 16px regular line height 1.5 
Expiration date copy 14px regular 
Price copy (within parter box) 16px semibold
Price total 18px semibold 
- please make sure text color is consistent with other styles in LMS

Button styling does not look consistent with other similar buttons within LMS

**Comment by clintonb** [comment]
@andy-armstrong I'd like to handle the general text/component styling in a separate PR, once we've determined which styles need to be ported from LMS.

**Comment by andy-armstrong** [comment]
@clintonb Was that just an FYI, or are you looking for advice and/or a review?

**Comment by clintonb** [comment]
@andy-armstrong FYI. I will ping you on the PR.

**Comment by andywaldrop** [comment]
@clintonb I'm guessing @andy-armstrong was meant for me?
#mistakenandy

**Comment by clintonb** [comment]
Yes, apologies to both Andy's/Andies.

**Comment by zubair-arbi** [comment]
@clintonb I have updated my PR to use bootstrap for much of scaffolding as you suggested.
@andywaldrop As @clintonb mentioned I used the basic colors and minimum styling until general styling is implemented. Here are two screen shots:
// medium-large devices
screen shot 2015-07-02 at 2 33 26 pm
// mobile devices
screen shot 2015-07-02 at 2 38 56 pm

File: ecommerce/static/sass/_credit.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    margin-top: 20px;
+
+    .payment-button {
+      width: 100%;
+    }
+  }
+}
+
+/* Small devices (tablets, 768px and up) */
+@media (min-width: 768px) {
```
Comment (clintonb) [inline ecommerce/static/sass/_credit.scss]: Use `@media (min-width: $grid-float-breakpoint-max)`

File: ecommerce/static/sass/main.scss (hunk 1 above)
Comment (clintonb) [inline ecommerce/static/sass/main.scss:23]: Move this import into base.scss.

**Comment by clintonb** [comment]
:+1: (once rebased, and remaining comments addressed)

**Comment by zubair-arbi** [comment]
@wedaly please review.

**Comment by wedaly** [comment]
Clinton gave it a :+1: , so go for it!

**Comment by zubair-arbi** [comment]
@wedaly I will merged it after replacing static content will real one.

**Pull request merged**